packages/dd-trace/src/format.js

```diff
@@ -64,6 +66,31 @@ function setSingleSpanIngestionTags (span, options) {
   addTag({}, span.metrics, SPAN_SAMPLING_MAX_PER_SECOND, options.maxPerSecond)
 }
 
+function extractSpanLinks (trace, span) {
+  const links = []
+  if (span._links) {
+    for (const link of span._links) {
+      const { context, attributes } = link
+      const formattedLink = {}
+
+      formattedLink.trace_id = context._traceId.toBuffer().length <= 8 && context?._trace?.tags['_dd.p.tid']
```

Comment:
Since this is the same as the code in `SpanContext`, ideally it would be in a single place. Maybe `toTraceId` could have a parameter to return this format?